docs/content/en/docs/Coding Milestones/PoC2023q1/outline.md

```diff
@@ -134,17 +162,40 @@ cluster, and the summarization part will affect the status summarizer.
 
 ## Design overview
 
-In very brief: the design is to reduce each edge placement problem to
-many instances of kcp's TMC problem.
+In very brief: the design is to reduce the edge placement problem to
+kcp's TMC problem, using as many "anycast" TMC Placement objects (each
```

Comment:
perhaps better `the design is to reduce` -> `the design goal is to reduce`

Reply:
Reworded in latest revision.